job_runner/worker.py

```
@@ -146,7 +170,22 @@ async def main():
+                    queue_name,
+                    timeout=QUEUE_POLL_TIMEOUT_SECONDS,
+                )
+            except RedisTimeoutError:
+                logger.debug("Redis queue poll timed out; waiting for the next poll")
+                continue
+            except RedisConnectionError as exc:
+                logger.warning("Redis connection lost while polling queue: %s", exc)
+                await redis_client.aclose()
+                await asyncio.sleep(REDIS_RECONNECT_DELAY_SECONDS)
+                redis_client = await create_redis_client()
+                logger.info(f"Local worker reconnected to Redis and is listening on {queue_name}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)